### ISSUE TITLE
How is uptime collected?

### ISSUE THREAD

**Issue by lonnietc** [YGGverse/YGGstate#4]
Hello,

I am exploring Yggdrasil for a project and environment and came across YGGstate which seems to get Uptime and location (meta)data about the active nodes.

I am wondering how this is collected and how to validate that the uptime reported for the nodes are accurate.

Do the peer nodes periodically send the data to YGGstate or is there some type of crawler that is active, or some other process?

Thanks and have a great day.

**Comment by d47081** [comment]
Hello, thanks for your question

YGGstate is mostly administrative tool with web UI for public peers connected, that also could be published for others on 80 port (with limited features), like

`http://[201:23b4:991a:634d:8359:4521:5576:15b7]/yggstate/` instance.

In this version, YGGstate receives uptime just by `yggdrasilctl getpeers` command, save the stats and visualize it. As peers connected to this node, their session could be tracked by without any verification (because public peer know how many traffic it have transmitted for every peer connected). 

In other words, current implementation of YGGstate can't track all the Yggdrasil network, but connected peers.
Some distributed features implemented to collect the data from different service nodes, just currently, user should use `tls://94.140.114.241:4708` public peer in [list](https://github.com/YGGverse/YGGstate#online-trackers) to be tracked.
